<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_interface</class>
 <widget class="QWidget" name="login_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>324</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>526</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="Layout_1">
     <property name="spacing">
      <number>22</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="QQ_number_label">
       <property name="text">
        <string>QQ号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="QQ_number_edit">
       <property name="placeholderText">
        <string>请输入QQ号</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>60</number>
     </property>
     <item>
      <widget class="QLabel" name="pass_label">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pass_edit">
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>请输入密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showpass_box">
       <property name="text">
        <string>显示密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_3">
     <property name="topMargin">
      <number>55</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="login_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="reset_button">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="register_button">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>新用户注册</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>48</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="forget_password">
       <property name="text">
        <string>忘记密码</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="reset_password">
       <property name="text">
        <string>修改密码</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="help">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
